gemrb/core/Scriptable/Actor.cpp

```diff
@@ -8219,9 +8221,25 @@ void Actor::SetRunFlags(ieDword flags)
 	InternalFlags |= (flags & IF_RUNFLAGS);
 }
 
+void Actor::NewPath()
+{
+        if (Destination == Pos) return;
+
+        Point tmp = Destination;
```

Comment:
I know it's not new, but now looking at it, it's not trivially clear why this temporary is needed -> please add a comment so someone doesn't accidentally remove it.